tests/core/test_farmer_harvester_rpc.py

```diff
@@ -171,14 +171,18 @@ async def have_plots():
 
             # Test farmer get_harvesters
             async def test_get_harvesters():
+                harvester.plot_manager.trigger_refresh()
+                await farmer_api.farmer.update_cached_harvesters(force_update=True)
```

Comment:
you don't think there's a risk that this might hide a real issue in the harvester?
I think it would be good to understand why the harvester's cache become stale. This won't happen in a real-world running of the harvester, right? and it needs to be able to pick up the plots anyway.

If you do understand why this fails and it would only ever happen in the test, I think there should be a comment here explaining it for future readers of this code.

Reply:
I don't think there is a risk. This just triggers the cache update earlier which happens usually with a 90s interval. I changed the code though to not have the force flag but instead just change the update interval.